### ISSUE TITLE
Accessibility: Incorrect heading order on Edit Address page

### ISSUE THREAD

**Issue by amberhinds** [woothemes/woocommerce#43633]
### Prerequisites

- [X] I have carried out troubleshooting steps and I believe I have found a bug.
- [X] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

The heading on the edit address page is set to H3s. The only other heading on the page is the "My Account" H1, so this skips a level.

https://github.com/woocommerce/woocommerce/blob/758df4854d7b627fc26eb5c0860dc492dea6a06e/plugins/woocommerce/templates/myaccount/form-edit-address.php#L30

### Expected behavior

Headings should create a hierarchical outline on the page and not skip levels. This heading should be coded as an H2 not H3.

### Actual behavior

Heading skip from H1 to H3.

### Steps to reproduce

See code linked above.

### WordPress Environment

N/A

### Isolating the problem

- [X] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [X] This bug happens with a default WordPress theme active, or [Storefront](https://woo.com/products/storefront/).
- [X] I can reproduce this bug consistently using the steps above.
